client/client_core.go

```diff
@@ -199,19 +207,41 @@ func (c *AnkaClient) Modify(vmName string, command string, property string, flag
 		log.Print("Error executing modify command: ", output.ExceptionType, " ", output.Message)
 		return fmt.Errorf(output.Message)
 	}
+
 	return nil
 }
 
-func (c *AnkaClient) Run(params RunParams) (error, int) {
+type RunParams struct {
+	VMName         string
+	Volume         string
+	Command        []string
+	Stdin          io.Reader
+	Stdout, Stderr io.Writer
```

Comment:
So rather than creating `RunWithOutput`, could we leverage the `stdout` here to capture the output from the normal `Run` command?